src/apps/AppCenter/AppCard.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Card,
+  GU,
+  IconExternal,
+  Tag,
+  textStyle,
+  unselectable,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import AppIcon from '../../components/AppIcon/AppIcon'
+
+const AppCard = React.memo(function AppCard({
+  onClick,
+  link,
+  icon,
+  name,
+  tag,
+  description,
+}) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+  const handleClick = useCallback(() => onClick(link), [onClick, link])
+
+  return (
+    <Card onClick={onClick} css={onClick ? 'display: block;' : ''}>
+      <CardMain compactMode={compactMode}>
+        {link && (
+          <StyledIconExternal
+            compactMode={compactMode}
+            theme={theme}
+            link={link}
+          />
+        )}
+        <Icon compactMode={compactMode}>{icon}</Icon>
+        <Name compactMode={compactMode}>{name}</Name>
+        <TagWrapper compactMode={compactMode} link={link}>
+          <Tag mode="indicator">{tag}</Tag>
```

Comment:
Ahh.. perhaps we can pass in a node here instead? For new versions, we should use `mode="new"`.

Reply:
Fixed: e9085905 , 497d2df7 and 0b59432d